### DISCUSSION TITLE
手写签名组件在弹窗内无法使用

### DISCUSSION THREAD

**Discussion by wl830** [dotnetcore/BootstrapBlazor#3873] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

在页面中使用SignaturePad组件可以正常使用，但是在弹窗中使用SignaturePad组件不生效

### Steps To Reproduce

[BootstrapBlazor.OnlyServer1.zip](https://github.com/user-attachments/files/16092866/BootstrapBlazor.OnlyServer1.zip)

### .NET Version

.NET 8

### Anything else?

![8d20881e4d24bdb1bb1b4004cdff2768](https://github.com/dotnetcore/BootstrapBlazor/assets/80016495/15ec0fd8-e7e2-43bb-8fe0-9bfbf1e16fca)
![69604981d58a4ef80e991a2a2210d90a](https://github.com/dotnetcore/BootstrapBlazor/assets/80016495/8649b7b5-6297-476e-9aa3-9d91e16fb6ca)

**Comment by bb-auto** [comment]
@wl830 Thank you for reporting. We will give triage later.

**Comment by densen2014** [comment]
请用另一个签名组件

**Comment by wl830** [comment]
> 请用另一个签名组件

测试过了，Handwritten 组件在弹窗中也不行

**Comment by densen2014** [comment]
不对啊，我写错了。 弹窗有demo啊

**Comment by wl830** [comment]
上面有复现的例子，可以运行试一下

**Comment by densen2014** [comment]
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/8428709/2dafa4a7-319e-4203-9f95-d85da00b8672)

首先，文档的你用起来有没有问题。

**Comment by densen2014** [comment]
试试这里 https://blazor.app1.es/signaturepadResponsive 的， bb文档格式可能有点问题要修复一下

**Comment by wl830** [comment]
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/80016495/94348ea1-9cee-4dad-846f-b9ebe09849b0)
这个不是弹窗形式的，而且背景是黑色的，看不到签名

**Comment by ArgoZhang** [comment]
@densen2014 可以把项目源码集成到 `src\Extensions\Components` 目录，我来弄弄

**Comment by densen2014** [comment]
下周一搞起

**Comment by ArgoZhang** [comment]
@wl830 @densen2014 发布了 `v8.0.2` 可根据上面的复现工程测试
